core/frontend/src/extension/ExtensionAdmin.ts

```diff
@@ -42,17 +26,12 @@ export class ExtensionAdmin {
    */
   public readonly onExtensionLoaded = new BeEvent<(extensionName: string) => void>();
 
-  public constructor(props?: ExtensionAdminProps) {
-    this._extensionAdminProps = props;
-  }
+  public constructor() { }
 
-  /** On view startup, [[IModelApp.viewManager.onViewOpen]], [[ExtensionAdmin]] will be setup according to the provided [[ExtensionAdminProps]].
+  /** On view startup, [[IModelApp.viewManager.onViewOpen]], [[ExtensionAdmin]] will be setup.
    * @beta
    */
-  public onInitialized() {
-    if (this._extensionAdminProps?.configureExtensionServiceLoader ?? true)
-      this.addExtensionLoaderFront(new ExtensionServiceExtensionLoader("00000000-0000-0000-0000-000000000000"));
-  }
+  public onInitialized() { }

```

Comment:
is there a reason to still keep this?

Reply:
I didn't want to remove it as I think we will need it when I go to implement the new way of loading Extensions. That's a large WIP PR that I'll hopefully have up for review by end of the week or early next week.